wordpress-mobile/WordPress-iOS
# WP.com abort login flow for self-hosted site

This PR fixes the issue where an issue was unable to abandon the WP.com login flow while logged into a self hosted site

Fixes #19245

To test:
1. Perform a fresh install of the app.  Confirm a cancel button is not visible on the initial login screen
2. Log into a self-hosted site using its site credentials 
3. Tap the account avatar to reach "Me" screen
4. Tap Log In under the "WordPress.com Account" section
5. You're taken to the WP.com log in screen 
6. Click Cancel button which will return you to "Me" screen
7. Test for WordPress app and Jetpack app
8. Test with both dark mode on and off. 

Screenshots with cancel button (fix) below
<image width="300" src="https://user-images.githubusercontent.com/88816724/192368687-521354d5-8112-439e-a701-e4a6189c6c30.png">

<image width="300" src="https://user-images.githubusercontent.com/88816724/192368764-ef844ecd-ad46-4c11-b7c6-029e0e55d93a.png">

<image width="300" src="https://user-images.githubusercontent.com/88816724/192368786-375aab0f-ff95-4889-8080-e59d92c080bb.png">


## Regression Notes
1. Potential unintended areas of impact
Login

2. What I did to test those areas of impact (or what existing automated tests I relied on)
I tested on device and simulator on WordPress and Jetpack apps including dark mode

3. What automated tests I added (or what prevented me from doing so)
None

PR submission checklist:
